python/perf-kernels/flash-attention.py

```diff
@@ -543,7 +663,12 @@ def forward(ctx, q, k, v, causal, bias, sm_scale, split_kernel=False):
         num_warps = 8 if Lq == 128 else 4
         pre_load_v = False if Lq == 128 else True
         grid = (triton.cdiv(q.shape[2], BLOCK_M), q.shape[0] * q.shape[1], 1)
-        
+
+        if return_encoded_softmax:
```

Comment:
Out in the world, there is very little detail on what exactly this is. It would be good to add information on that here as comment, if we know.